pkg/runtime/from_config.go

```diff
@@ -21,12 +21,14 @@ import (
 var (
 	regCrd  *regexp.Regexp
 	regURL  *regexp.Regexp
+	regGRPC *regexp.Regexp
 	regFile *regexp.Regexp
 )
 
 func init() {
 	regCrd = regexp.MustCompile("^core.openfeature.dev/")
 	regURL = regexp.MustCompile("^https?://")
+	regGRPC = regexp.MustCompile("^" + grpc.Prefix)
```

Comment:
Probably a question for another day / PR... but how do we handle authentication, especially if we have multiple gRPC syncs with different credentials?

I'll also leave this [here](https://grpc.io/docs/guides/auth/) to plug my mutual TLS suggestion :sweat_smile:

Reply:
I am actively working on a follow-up PR to introduce secure connectivity based on TLS. Haven't considered tokens yet though.

> but how do we handle authentication, especially if we have multiple gRPC syncs with different credentials?

This is the challenge. At least for the initial version, I am considering having an `enable security` configuration along with a `certificate path` to support self-signed certificates. This will be applicable for "all"  grpc connections though. 

May be once that's in place, we can think about `key:value` based solution where key helps to identify specific grpc target and use specific value and so on. IMO a conversation for the next pr 😉